src/api.calls.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios'
+import type {
+  PaginationSizeTypes,
+  PokemonCollectionResponse,
+  SinglePokemonResponse
+} from './models/pokedex'
+const pokemonURL = `https://pokeapi.co/api/v2/pokemon`
+
+export async function getPokemon(size: PaginationSizeTypes, page: number) {
+  try {
+    let reqUrl = '?'
+    if (size === 'All') {
+      reqUrl +=  'limit=' + 1302 
+    }
+    else {
+      reqUrl += 'limit=' + size + '&offset=' + size * page
+    }
+    const result = await axios.get<PokemonCollectionResponse>(`${pokemonURL}/${reqUrl}`)
```

Comment:
You can just directly return the await here since result isn't used besides returning